### DISCUSSION TITLE
Update for iTwin release 3.0

### DISCUSSION THREAD

**Discussion by monikagole** [iTwin/itwinjs-core#3026] [Q&A] [answered]
1. Is iTwin version 3.0 released publicly?
2. Is V3.0 released internally? If yes, then how can I get access to V3.0?

Any suggestions on how I can access iTwin version 3.0?

**Comment by pmconne** [comment, marked as answer]
See the [pinned announcement](https://github.com/iTwin/itwinjs-core/discussions/2942).
